pydsa/dfs.py

```diff
@@ -0,0 +1,16 @@
+# Depth First Search
+# Complexity: O(V+E)
+#uses concept of list
+
+def dfsearch(graph,s):
```

Comment:
it is working. was missing something. but still importing is different than bfs. don't know why. Will push the code in evening.